### PR TITLE
Improve some logging in cacher

### PR DESCRIPTION
**Pull request by wojtek-t**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/34322)

### CODE REVIEW

**Comment by gmarek** [comment]
One thing. LGTM otherwise.

File: pkg/storage/cacher.go [hunk 1]
```diff
@@ -680,8 +680,13 @@ func (c *cacheWatcher) add(event *watchCacheEvent) {
...
 		c.stop()
 	}
-	glog.V(2).Infof("cacheWatcher add function blocked processing of %v for %v (initial result size %v)",
-		reflect.TypeOf(event.Object).String(), time.Since(startTime), resultLen)
+
+	const waitingThreshold = 5 * time.Millisecond
+	waitingTime := time.Since(startTime)
+	if waitingTime > waitingThreshold {
+		glog.V(2).Infof("cacheWatcher add function blocked processing of %v for %v (initial result size %v)",
+			reflect.TypeOf(event.Object).String(), time.Since(startTime), resultLen)
```
Comment (wojtek-t) [inline pkg/storage/cacher.go]: Discussed offline and changed to trace.

**Comment by wojtek-t** [comment]
@gmarek - PTAL

**Comment by gmarek** [comment]
LGTM

**Pull request merged**